index.bs

```diff
@@ -9844,19 +9849,134 @@ Constructors</h5>
 <dl dfn-type=constructor dfn-for="AudioWorkletNode">
 	: <dfn>AudioWorkletNode(context, name, options)</dfn>
 	::
-		Let <var>node</var> be a new {{AudioWorkletNode}} object.
-		<a href="#audionode-constructor-init">Initialize</a>
-		<var>node</var>. Perform the <a href="#instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
-		construction procedure</a> of an
-		{{AudioWorkletNode}} and the corresponding
-		{{AudioWorkletProcessor}} object. Return
-		<var>node</var>.
+		When the constructor of {{AudioWorkletNode}} is invoked in the
+		main global scope, the following construction algorithm MUST be
+		performed.
+
+		During the construction of an AudioWorkletNode, a
```

Comment:
In line 9853 you say "algorithm".  It would be nice if this paragraph were restructured to be an algorithm with steps.  But it's ok with me if you don't want to do that. (I'm not sure how to make it an algorithm with steps.)

Reply:
Oh. Maybe move the argument block further up or down so that the algorithm steps are closer to this paragrph.